examples/basic-example/main.tf

```diff
@@ -0,0 +1,53 @@
+
+module "this" {
+  source = "../../"
+
+  region               = var.region
+  unique_suffix        = var.unique_suffix
+  kms_key_arn          = var.kms_key_arn
+  server_domain        = var.website_domain
+  mqtt_broker_endpoint = var.mqtt_broker_endpoint
+
+  license_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQgeW91IHJlYWxseSB0aGluayB3ZSdsbCBhZGQgYSB2YWxpZCB0b2tlbiBoZXJlPyA6KSJ9.I60CMAP8z_ULiJgKMRrJqHgmYaiRm8PjSJh_l3AoCrU"
```

Comment:
Damn, you got me `{"sub":"did you really think we'll add a valid token here? :)"}`

Reply:
<img width="316" alt="image" src="https://github.com/user-attachments/assets/88c6c93a-baf4-406b-b01d-35aac1627ea0" />